tutorials/3d/environment_and_post_processing.rst

```diff
@@ -391,11 +598,149 @@ and Saturation:
 
 .. image:: img/environment_adjustments_bcs.png
 
-The second is by supplying a color correction gradient. A regular black to
-white gradient like the following one will produce no effect:
+The second is by supplying a color correction gradient. This can be done by
+assigning a GradientTexture1D resource to the **Color Correction** property, or
+by loading a texture containing a horizontal gradient. The leftmost of the
+gradient represents black in the source image, whereas the rightmost part of the
+gradient represents white in the source image.
 
-.. image:: img/environment_adjusments_default_gradient.png
+A linear black-to-white gradient like the following one will produce no effect:
+
+.. image:: img/environment_adjustments_default_gradient.png
 
 But creating custom ones will allow to map each channel to a different color:
 
-.. image:: img/environment_adjusments_custom_gradient.png
+.. image:: img/environment_adjustments_custom_gradient.png
+
+A 3D look-up-texture (LUT) can also be used for color correction. This is a
+special texture used to modify each color channel separately from one another
+(red, green, blue). This image can be of any resolution, but since color
+correction is low-frequency data, sticking to low resolutions is recommended for
+performance reasons. A LUT texture's resolution is typically 17×17×17, 33×33×33,
+51×51×51 or 65×65×65 (the odd size allows for better interpolation).
+
+.. note::
+
+    Color data within the LUT must be stored in linear space (not
+    perceptual/sRGB), as Godot performs 3D rendering in linear space.
```

Comment:
This is misleading, color-correction happens in perceptual/sRGB space in Godot. 
https://github.com/godotengine/godot/blob/ea47e03b360d0aeee10c1f7ef798f5750c2b1aa8/servers/rendering/renderer_rd/shaders/effects/tonemap.glsl#L491-L493

3D rendering does happen in linear space but color correction happens after the linear->sRGB conversion in the tonemapping pass

Reply:
Should I remove the note entirely? I'm not sure how to cause the least confusion here.

Reply:
Yes, might as well remove the note entirely